services/codepipeline/src/it/java/software/amazon/awssdk/services/codepipeline/AwsCodePipelineClientIntegrationTest.java

```diff
@@ -45,7 +52,46 @@
     @BeforeClass
     public static void setup() throws Exception {
         setUpCredentials();
-        client = CodePipelineClient.builder().credentialsProvider(CREDENTIALS_PROVIDER_CHAIN).build();
+
+        ClientOverrideConfiguration configuration = ClientOverrideConfiguration
+            .builder()
+            .metricConfigurationProvider(DefaultMetricConfigurationProvider.builder()
+                                                                           .enabled(true)
+                                                                           .build())
+            .metricPublisherConfiguration(MetricPublisherConfiguration.builder().build())
+            .build();
+
+        client = CodePipelineClient.builder()
+                                   .credentialsProvider(CREDENTIALS_PROVIDER_CHAIN)
+                                 //  .overrideConfiguration(configuration)
```

Comment:
can be removed?

Reply:
Need to look this class over again. Probably not worthwhile to include until publisher is added